packages/notification-center-angular/projects/notification-center-angular/README.md

```diff
@@ -16,7 +16,7 @@ This library contains a wrapper for the Novu Notification Center web component,
 
 ## 📖 Client Installation
 
-For our client installation guide, visit our [Angular Client docs](https://docs.novu.co/notification-center/client/angular).
+For our client installation guide, visit our [Angular Client docs](https://docs.novu.co/notification-center/client/angular?utm_campaign=github-nc-readme).
```

Comment:
It'll be worthwhile to be specific about which NC framework is used here, because we have a few already and this will grow (React, Angular,...)

```suggestion
For our client installation guide, visit our [Angular Client docs](https://docs.novu.co/notification-center/client/angular?utm_campaign=github-notificationcenter-angular-readme).
```